docs/pages/visual-test/DatePicker-closed-dark.js

```diff
@@ -0,0 +1,14 @@
+// @flow strict
+import type { Node } from 'react';
+import { Box, ColorSchemeProvider } from 'gestalt';
+import GestaltDatePicker from 'gestalt-datepicker';
+
+export default function DatePickerSpec(): Node {
+  return (
+    <ColorSchemeProvider colorScheme="dark">
```

Comment:
Hm…I wonder for consistency's sake if we should include this in the light mode tests as well? I have no idea what the correct answer is, looking for opinions

Reply:
I actually did that in the Icon one. I'll add it here.